crates/api/src/chronicle_graphql/mod.rs

```diff
@@ -72,6 +72,11 @@ pub struct Identity {
 }
 
 #[Object]
```

Comment:
I haven't figured out how to use `include_str`the way I've been able to in `codegen` for these types that get included in GraphQL schema.

Reply:
Does desc / description work in the macro body rather than as a comment? https://docs.rs/async-graphql/latest/async_graphql/attr.Object.html

Reply:
I also tried the following but I get the same `expected literal` error that I haven't been able to figure out.

```rust
#[Object]
#[graphql(desc = include_str!("../../../../domain_docs/identity.md"))]
impl Identity {
    async fn public_key(&self) -> &str {
        &self.public_key
    }
}
```

The following results in the same issue, whereas it works on "normal" Rust code: 

```rust
#[Object]
#[doc = include_str!("../../../domain_docs/identity.md")]
impl Identity {
    async fn public_key(&self) -> &str {
        &self.public_key
    }
}
```

Experimenting with the place of the `///` doc comment showed it would only show up in the GraphQL schema where it is now.

Reply:
Same if you do that:

```rust
#[Object(desc = include_str!("../../../../domain_docs/association.md"))]
```

Reply:
Annoying!